packages/backend/src/services/GitIntegrationService/GitIntegrationService.ts

```diff
@@ -439,6 +445,7 @@ Affected charts:
             owner,
             customMetrics,
             repo,
+            path,
```

Comment:
I also changed this, but I believe this method `createPullRequestForChartFields` is not used, we should probably remove it